app/src/androidTest/java/com/github/houseorganizer/houseorganizer/MainScreenActivityTest.java

```diff
@@ -160,4 +163,25 @@ public void calendarViewRotatesCorrectly() {
 
     // TODO : Add more meaningful tests for each row in the RecyclerViews (no idea how to do it)
 
+
+    @Test
+    public void signOutButtonIsDisplayedAndEnabled(){
+        onView(withId(R.id.sign_out_button)).check(matches(isDisplayed()));
+        onView(withId(R.id.sign_out_button)).check(matches(isEnabled()));
```

Comment:
Separate the two tests for displayed and enabled. A button can be display but disabled at some later point in time during the execution of the app.